shell/src/main/java/alluxio/cli/fs/command/StatCommand.java

```diff
@@ -61,6 +63,11 @@ public Options getOptions() {
             .hasArg()
             .desc("format")
             .build()
+    ).addOption(
+        Option.builder("fileId")
+            .required(false)
+            .desc("fileId")
```

Comment:
More detailed description for this new option?

Reply:
done